fiftyone.yml

```diff
@@ -0,0 +1,6 @@
+fiftyone:
+  version: ~0.21.0
+name: '@voxel51/fiftyone-docs-search'
```

Comment:
I think we should encourage this pattern, which is to match the github repo name to the plugin name. In theory we could also split this and then every "name" in this file is technically prefixed with the org name:

```yaml
org: voxel51
name: fiftyone-docs-search
operators:
  - name: search-docs
```

Reply:
I prefer t

> I think we should encourage this pattern, which is to match the github repo name to the plugin name. In theory we could also split this and then every "name" in this file is technically prefixed with the org name:
> 
> ```yaml
> org: voxel51
> name: fiftyone-docs-search
> operators:
>   - name: search-docs
> ```

I prefer this version. Typing `@voxel51/fiftyone-docs-search` for plugin name in zoo methods is an unnecessary pain